xml/System.Text/Rune.xml

```diff
@@ -1032,9 +1032,9 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <Parameter Name="value" Type="System.UInt32" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The 32-bit signed integer to convert.</param>
+        <summary>Defines an explicit conversion of a 32-bit unsigned integer to a <see cref="T:System.Text.Rune"/>.</summary>
+        <returns><paramref name="value" /> as a 32-bit Unicode character.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Exception for an invalid Unicode scalar value.

Reply:
```suggestion
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="value" /> is an invalid Unicode scalar.</exception>
```